packages/buefy-next/src/components/progress/ProgressBar.vue

```diff
@@ -17,10 +17,14 @@
     </div>
 </template>
 
-<script>
-import { PROGRESS_INJECTION_KEY } from './Progress.vue'
+<script lang="ts">
+import { defineComponent } from 'vue'
+import type { ExtractComponentProps } from '../../utils/helpers'
+import Progress, { PROGRESS_INJECTION_KEY } from './Progress.vue'
 
-export default {
+type ProgressInstance = InstanceType<typeof Progress>
+
+const ProgressBar = defineComponent({
     name: 'BProgressBar',
     inject: {
```

Comment:
> @ntohq/buefy-next@0.1.4 type-check
> vue-tsc --noEmit -p tsconfig.test.json --composite false

src/components/progress/ProgressBar.vue:7:25 - error TS2551: Property 'parent' does not exist on type '{ $: ComponentInternalInstance; $data: {}; $props: Partial<{ type: string; value: number; showValue: boolean; }> & Omit<{ readonly showValue: boolean; readonly type?: string | undefined; readonly value?: number | undefined; } & VNodeProps & AllowedComponentProps & ComponentCustomProps & Readonly<...>, DefaultKeys<.....'. Did you mean '$parent'?

7         :aria-valuemax="parent.max"
                          ~~~~~~

  ../../node_modules/@vue/runtime-core/dist/runtime-core.d.ts:122:5
    122     $parent: ComponentPublicInstance | null;
            ~~~~~~~
    '$parent' is declared here.


Found 1 error in src/components/progress/ProgressBar.vue:7